### PR TITLE
added reference to external example flutter app

### PR DESCRIPTION
**Pull request by wolfc01**
Hello

@werediver @remonh87, as a follow up of https://github.com/PhilipsHue/flutter_reactive_ble/issues/170, and pull request https://github.com/PhilipsHue/flutter_reactive_ble/pull/178, I created a repository containing the example.

As a reminder, I created this pull request adding an reference to it in the READ.md file. I'm happy to receive review comments on this example.

Br
Carl.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -217,3 +217,7 @@ In case you are using ProGuard add the following snippet to your `proguard-rules
 ```
 
 This will prevent issues like [#131](https://github.com/PhilipsHue/flutter_reactive_ble/issues/131) .
+
+#### Further unofficial examples
```
Comment (remonh87) [inline README.md:221]: I would name it unofficial example apps

File: README.md [hunk 2]
```diff
@@ -217,3 +217,7 @@ In case you are using ProGuard add the following snippet to your `proguard-rules
...
+
+An example implementing an UART over BLE can be found at https://github.com/wolfc01/flutter_reactive_ble_uart_example
```
Comment (remonh87) [inline README.md:223]: Example implementation UART over BLE: link

Use the markdown for links `[description](url)`

**Review by remonh87 — COMMENTED** [review]
In general looks good maybe check these small textual changes, then I will merge it in. Thanks :)

**Comment by wolfc01** [comment]
Hi,

review comments processed....

Br
Carl.

**Review by remonh87 — APPROVED** [review]

**Pull request merged**